src/io/_Private/LegacyPHPResourceSeekableHandleTrait.php

```diff
@@ -0,0 +1,32 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\IO\_Private;
+
+use namespace HH\Lib\{Experimental\IO, Str};
+
+trait LegacyPHPResourceSeekableHandleTrait implements IO\SeekableHandle {
+  require extends LegacyPHPResourceHandle;
+  /**
+   * Move to a specific offset within a handle.
+   *
+   * Offset is relative to the start of the handle - so, the beginning of the
+   * handle is always offset 0.
+   *
+   * Any other pending operations (such as writes) will complete first.
+   */
```

Comment:
not related to this PR but it would be nice to have some mechanism for implementations to inherit the docblock from the interface (or link to it somehow)

Reply:
```php
/**
 * {@inheritdoc}
 */
```